README.rst

```diff
@@ -21,8 +21,8 @@ RENGA (連歌)
 .. image:: https://img.shields.io/travis/SwissDataScienceCenter/renga.svg
    :target: https://travis-ci.org/SwissDataScienceCenter/renga
 
-.. image:: https://readthedocs.org/projects/renga/badge/?version=latest
```

Comment:
I would suggest to keep all on latest and merge development to master ASAP, keeping the old master as `maint-0.1` (cc @erbou)

Reply:
that's fine with me -- we should have a conclusion on #138 before doing this I suppose